### DISCUSSION TITLE
Unable to install nxt-editor for Blender

### DISCUSSION THREAD

**Discussion by sgodanas** [nxt-dev/nxt_editor#210] [Q&A] [answered]
OS: Windows 10
Blender Version: 2.93.4

Hello Folks.

I am getting the following error. Any ideas? 

```
PS C:\pipeline\software\blender\incoming\stable>> & 'C:\Program Files\Blender Foundation\Blender 2.93\2.93\python\bin\python.exe' -m pip install nxt-editor
ERROR: Could not find a version that satisfies the requirement nxt-editor (from versions: none)
```

It works fine for Blender 2.92.

Thanks in advance.

Satish.

**Comment by ImLucasBrown** [comment, marked as answer]
Looks like Blender now ships, with Python `3.9` but we only _officially_ support up to `3.7`. The last time we tested installing the blender plugin it was in Blender 2.91. 
We make no promises any of this will work, but you can try one of the following:
- `pip install nxt-editor --ignore-requires-python` you'll probably have also individually install `PySide2`, `Qt.py` and `nxt-core`
- Change the python version requirement in our [setup.py](https://github.com/nxt-dev/nxt/blob/release/setup.py) and run our [package builder graph](https://github.com/nxt-dev/nxt/blob/release/build/packaging.nxt)

Supporting newer versions of Python is something we'd like to do, but probably won't happen this year.

**Comment by sgodanas** [comment]
Thank you @ImLucasBrown. I will try out your answer and let you know how it goes.
